test/cwl_util.go

```diff
@@ -59,6 +58,14 @@ func ValidateLogs(t *testing.T, logGroup, logStream string, numExpectedLogs int,
 		output, err = cwlClient.GetLogEvents(*clientContext, params)
 
 		if err != nil {
+			var rnf *types.ResourceNotFoundException
+			if errors.As(err, &rnf) {
```

Comment:
Do we need a default max retry count before giving up?

Reply:
Ah, you're right. I can fix that. I think 5 is a good round number. If the log group doesn't exist by then, something is up